### ISSUE TITLE
Reset not working?

### ISSUE THREAD

**Issue by cichecki** [jlmakes/scrollreveal.js#250]
Hi Julian and other guys,

I'm pretty newbie to coding and especially to JS so maybe my question would be a bit dumb, but I can't handle it.

I'm creating wordpress site using zerif lite theme ( https://pl.wordpress.org/themes/zerif-lite/ ). By default animations made by Scrollreveal are not repeating and I'm trying to change it. I found some issues here about this ubject but nothing really worked. 

Example of code in testimonials.php that I tried to change: 

```php
echo '<div class="row" data-scrollreveal="enter right after 0s over 1s">';
```

I tried adding `reset` there but it didn't make any change. 

```php
echo '<div class="row" data-scrollreveal="enter right after 0s over 1s and reset">';
echo '<div class="row" data-scrollreveal="enter right after 0s over 1s, reset">';
```

Am I doing it in wrong way or am I editing wrong part of code?

Also maybe you could tell me where's the difference between this two examples:

https://scrollrevealjs.org/
https://codepen.io/samarkandiy/pen/rvACy

They work the same way except resetting animation. Is there a quick and easy way to change the code of codepen example so it would work as demo on scrollrevealjs.org?

Thanks!

**Comment by jlmakes** [comment]
Welcome to GitHub @cichecki,

#### Your Questions

>I'm pretty newbie to coding and especially to JS so maybe my question would be a bit dumb, but I can't handle it...

There are no dumb questions. I’ve been at this for 15 years, and trust me, there are always moments when you're just like...

![imgres](https://cloud.githubusercontent.com/assets/2044842/14685146/a24de896-0733-11e6-8c4a-11241277b621.jpg)

>...maybe you could tell me where's the difference between this two examples:

Sure.

>https://scrollrevealjs.org/

This is the new demo page, using ScrollReveal `v3.1.4`, redesigned and launched Dec 15th, 2015.

>https://codepen.io/samarkandiy/pen/rvACy

This appears to be the _very first_ release of ScrollReveal `v0.0.1` which released almost 2.5 years ago on Jan 22nd, 2014! Avaz Bokiev authored that CodePen 2 days later, which is a stripped down version of the [original demo page](https://scrollrevealjs.org/legacy).

>Is there a quick and easy way to change the code of codepen example so it would work as demo on scrollrevealjs.org?

No. The `reset` feature wasn’t introduced until version `0.0.2`, released about 2 weeks later on Feb13th, 2014. (I’m actually tickled you found this demo, as I no longer had the original source at first launch.)

>Am I doing it in wrong way or am I editing wrong part of code?

I can already tell from your sample snippets, that you’re using a legacy version of ScrollReveal.

- Version 3 is the current and supported version of ScrollReveal.
- Version 2 is unsupported, but its [documentation is still available in the Wiki](https://github.com/jlmakes/scrollreveal.js/wiki)
- Version 1 (public beta) is completely unsupported.

#### Recommendation

I don’t know what sort of integration the author of the Zerif theme has made, but the first step would be to find out what version of ScrollReveal is actually baked in.

If there’s a cool integration into the admin panel for example, it might be nice to attempt an upgrade to a slightly newer legacy version (i.e. `0.1.3` or `2.3.2`), but...

In all likelihood, the best path is to just remove the ScrollReveal that is packaged into the theme download and all references to the `data-scrollreveal` attribute... and just install/use version 3 yourself.

**Comment by cichecki** [comment]
Thanks for the answer @jlmakes !

I finally managed to make it work like I wanted. Uff..

Thanks for help!
